markdown.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Markdown
+description: Markdown is a lightweight markup language which is used to create formatted text using simple text editor.
+created: 2022-10-05
+updated: 2022-10-05
+---
+ 
+#### Github supports Markdown and thats why we are going to create this documentation in .md file.
+
+* Content:
+  - Basic syntax.
+  - Advanced syntax.
+  
+![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/rainbow.png)
```

Comment:
Can you use OneCompiler images?

Reply:
I have changed it. Kindly review and merge it.